flow/src/main/scala/org/alephium/flow/network/DiscoveryServer.scala

```diff
@@ -135,7 +135,12 @@ class DiscoveryServer(
       unstashAll()
       log.debug(s"bootstrap nodes: ${bootstrap.mkString(";")}")
       startBinding()
-      scheduleOnce(self, InitialDiscoveryDone, discoveryConfig.initialDiscoveryPeriod)
+
+      if (networkConfig.networkId == NetworkId.AlephiumMainNet) {
+        scheduleOnce(self, InitialDiscoveryDone, discoveryConfig.initialDiscoveryPeriod)
+      } else {
+        scheduleOnce(self, InitialDiscoveryDone, Duration.ofSecondsUnsafe(1))
+      }
```

Comment:
but what if I'm developing on devnet and I want to have a different discovery period?

Reply:
scanning would still go on. The initial discovery is about when we feel it's enough to connect to those discovered peers: https://github.com/alephium/alephium/blob/master/flow/src/main/scala/org/alephium/flow/network/DiscoveryServer.scala#L201-L204

Reply:
ok :+1: